### PR TITLE
Support 3p iframe addressing in the local mode

### PR DESCRIPTION
**Pull request by dvoytenko**
Includes `<meta name="amp-3p-iframe-src">` in the local mode to address 3p iframe correctly.

### CODE REVIEW

File: src/register/components/PreactDecorator.tsx [hunk 1]
```diff
@@ -155,6 +156,15 @@ function getBase(config: Config): string {
   return `<base href="${new URL(config.baseUrl).origin}">`;
 }
 
+function get3pIframeMeta(config: Config): string {
+  if (!config.baseUrl.startsWith("http://localhost")) {
```
Comment (caroqliu) [inline src/register/components/PreactDecorator.tsx:160]: Should we do anything for PR deploy bot domains too?

**Review by caroqliu — APPROVED** [review]

**Review by alanorozco — APPROVED** [review]

File: src/register/components/PreactDecorator.tsx (hunk 1 above)
Comment (dvoytenko) [reply src/register/components/PreactDecorator.tsx:160]: I don't know how those work, TBH, but I think something should be done there as well. @alanorozco can you explain how that works?

Comment (dvoytenko) [reply src/register/components/PreactDecorator.tsx:160]: I'll merge and we'll follow up on this subject separately.

Comment (alanorozco) [reply src/register/components/PreactDecorator.tsx:160]: We should probably sync offline because I don't know how it works on `localhost` 🙂

**Pull request merged**